### DISCUSSION TITLE
Option to include cells with 0 counts for the target in pairwise QC

### DISCUSSION THREAD

**Discussion by redbybeing** [Katsevich-Lab/sceptre#111] [Q&A] [answered]
Hi!

As far as I understand, in QC step, for a given gene-gRNA pair, sceptre removes cells (both ctrl & perturbed) where the target gene count is 0.
Our team found that for our dataset, there is a great need for including cells with 0 counts for the target gene in `run_qc()`.
Is there an easy way to do that?

Thank you so much!

Best,
Jiseok

**Comment by timothy-barry** [comment, marked as answer]
Hi Jiseok,

This is not quite accurate. `sceptre` does not remove cells as a function of target-response pair. The [cellwise QC step](https://timothy-barry.github.io/sceptre-book/run-qc.html) removes a fixed set of cells irrespective of the target-response pairs that are tested.

 It seems to me like you are running into issues with pairwise QC as opposed to cellwise QC. My recommendation would be to play around with the `n_nonzero_trt` and `n_nonzero_cntrl` parameters. In particular, I'd recommend setting `n_nonzero_trt` to 0.

**Comment by redbybeing** [comment]
Hi Tim,

I'm sorry for the late reply- I was swept away with things. 
Yes I am interested in maintaining pairs where the response is 0. I will try your suggestions!

Best,
Jiseok
